server/routerlicious/packages/services-ordering-rdkafka/.eslintrc.cjs

```diff
@@ -18,4 +18,13 @@ module.exports = {
 		// TODO: remove usages of deprecated APIs and remove this override
 		"import/no-deprecated": "warn",
 	},
+	overrides: [
+		{
+			files: ["src/rdkafkaBase.ts"],
+			rules: {
+				// It's fine for tests to use node.js modules.
```

Comment:
We should move the rule-off to `rules` above and get rid of overrides for a few reasons: a) server-only package, so it's fine for it to use node modules, and b) the node module is not being used in a _test_ file, so the comment in the previous line would be wrong.